database/migrations/2024_07_25_173451_make_remanded_fields_nullable_in_victims_table.php

```diff
@@ -0,0 +1,22 @@
+<?php
```

Comment:
@Marshud Merge this migration into the previous one  by just adding the nullable trait, it will be broken for existing data

Reply:
@ssmusoke the PR has been updated as per your recommendations